src/components/molecules/NavBar/NavBar.scss

```diff
@@ -246,16 +195,95 @@ $border-radius: 28px;
   &__home-icon {
     cursor: pointer;
     font-size: 24px;
+    color: opaque-white(0.5);
 
     margin-right: $spacing--md;
+  }
 
-    &:hover {
-      font-size: 27px;
-      margin-right: $spacing--sm;
+  &__separator {
+    margin: 0 $spacing--xs 0 $spacing--xs;
+  }
+
+  &__sparkle {
+    &-logo {
+      margin-right: $spacing--lg;
+      @include square-size($logo-size);
+      background-image: url(/assets/icons/sparkle-nav-logo.png);
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: center;
     }
   }
 
   &__profile-popover {
     height: 100%;
   }
+
+  &__radio {
+    &-container {
+      padding: $spacing--lg $spacing--md;
+      margin-top: $spacing--lg;
```

Comment:
![image](https://user-images.githubusercontent.com/63313732/130797141-5218235b-7814-4fad-9489-ca6a89daed30.png)

we can leave just padding.

i think we don't need that margin. it's too much of free space on top of the modal.